docs/user_guide/hopsworks/featurestore.rst

```diff
@@ -504,17 +504,42 @@ In the AWS management console go to *IAM*, select *Roles* and then the role that
+
+.. _hopsworks_parameter_store.png: ../../_images/parameter_store.png
+.. figure:: ../../imgs/feature_store/parameter_store.png
+    :alt: Hopsworks feature store parameter store
+    :target: `hopsworks_parameter_store.png`_
+    :align: center
+    :scale: 20 %
+    :figclass: align-center
+
+**(Alternative 2) Granting access to the secret to the SageMaker notebook role**
+
+In the AWS management console go to *IAM*, select *Roles* and then the role that is used when creating SageMaker notebook instances. Select *Add inline policy*. Choose *Systems Manager* as service, expand the *Read* access level and check *GetParameter*. Expand Resources and select *Add ARN*. Fill in the region of the *Systems Manager* as well as the name of the parameter **WITHOUT the leading slash** e.g. *hopsworks/project/[MY_HOPSWORKS_PROJECT]/role/[MY_SAGEMAKER_ROLE]/type/api-key* and click *Add*. Click on *Review*, give the policy a name und click on *Create policy*.
+
+.. _hopsworks_aws_policy2.png: ../../_images/aws_policy2.png
+.. figure:: ../../imgs/feature_store/aws_policy2.png
+    :alt: Hopsworks feature store set policy
+    :target: `hopsworks_aws_policy2.png`_
+    :align: center
+    :scale: 30 %
+    :figclass: align-center
+
+**Installing hopsworks-cloud-sdk and connecting to the Feature Store**
+
+To be able to access the Hopsworks Feature Store the hopsworks-cloud-sdk library needs to be installed. One way of achieving this is opening a Python notebook in SageMaker and installing the latest hopsworks-cloud-sdk. Note that the library will not be persistent. For information around how to permanently install a library to Sagemaker see `Install External Libraries and Kernels in Notebook Instances <https://docs.aws.amazon.com/sagemaker/latest/dg/nbi-add-external.html>`_. ::
+
+    !pip install hopsworks-cloud-sdk
 
 You can now connect to the Feature Store::
 
     import hops.featurestore as fs
-    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project')
+    fs.connect('my_instance.us-east-2.compute.amazonaws.com', 'my_project', secrets_store = 'secretsmanager')
 
-If you have trouble connecting, then ensure that the Security Group of your Hopsworks instance on AWS are configured to allow incoming traffic from your SageMaker instance. See `VPC Security Groups <https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html>`_. for more information.
+If you have trouble connecting, then ensure that the Security Group of your Hopsworks instance on AWS are configured to allow incoming traffic from your SageMaker instance. See `VPC Security Groups <https://docs.aws.amazon.com/vpc/latest/userguide/VPC_SecurityGroups.html>`_. for more information. If you Sagemaker instances are no in the same VPC as your Hopsworks instance and the Hopsworks instance is not accessible from the internet then you will need to configure `VPC Peering on AWS <https://docs.aws.amazon.com/vpc/latest/peering/what-is-vpc-peering.html>`_.
```

Comment:
@steffengr instance -> instances or are - > is
no -> not

Reply:
done